### ISSUE TITLE
After Adding Image , We cant add Content In Side which is left blank. If i have small image i left align it , then i want to type in something beside image left space . We cant do that !

### ISSUE THREAD

**Issue by Sanjay007** [jpuri/react-draft-wysiwyg#770]
After Adding Image , We cant add Content In Side which is left blank. If i have small image i left align it , then i want to type in something beside image left space . We cant do that !

<img width="592" alt="wisyg" src="https://user-images.githubusercontent.com/5675688/50995118-212bf880-1544-11e9-8d33-ed703d296861.png">

**Comment by jpuri** [comment]
Hi @Sanjay007 , that is a limitation with the editor :(
But hard to fix with DraftJS.

**Comment by Sanjay007** [comment]
but i see  its done somehow in medium , not sure what they use ! i guess if we can handle click event and instert an empty tag

**Comment by jpuri** [comment]
Yep some advanced editor like medium have this feature of inline editor, but unfortunately for this one :(
The editor is highly customisable, you can implement it as controlled component and add this. I will also be happy to have a nice PR on this.

**Comment by ForteDell** [comment]
Actually doable with css
.editor-class figure {
	margin: 0;
}

.fc-richtext-editor {
	//this allows lists next to images
	ol {
		display: table; //IE/Edge
		display: flow-root;
	}

	ul {
		display: table; //IE/Edge
		display: flow-root
	}
	.rdw-image-left {
		display: unset;
		float: left;
		margin: 1em;
	}

	.rdw-image-right {
		display: unset;
		justify-content: unset;
		float: right;
		margin: 1em;
	}

	.rdw-image-center {
		margin: 1em;
	}

	//allow styling  to match segment in card
	.public-DraftStyleDefault-block {
		margin: unset;
	}

	//ensures hover on figure is always possible
	.rdw-image-imagewrapper {
		position: relative;
		z-index: 101;
	}

	//Overwrite all pasted fonts to our 'preferred' fonts
	.DraftEditor-editorContainer {
		span {
			font-family: Arial, Helvetica, "Helvetica Neue", sans-serif;
		}
	}
}

**Comment by kishan-tocca** [comment]
> After Adding Image , We cant add Content In Side which is left blank. If i have small image i left align it , then i want to type in something beside image left space . We cant do that !
> 
> <img alt="wisyg" width="592" src="https://user-images.githubusercontent.com/5675688/50995118-212bf880-1544-11e9-8d33-ed703d296861.png">

facing same issue

**Comment by arik555** [comment]
@jpuri  @Sanjay007 Hello. 
We can do it using CSS.

**The code is given below.** 

.rdw-image-alignment {
  display: inline !important;
}

.rdw-image-center {
  display: flex !important;
  justify-content: center;
}

.rdw-image-right {
  float: right !important;
}

.rdw-image-left {
  float: left !important;
}

![image](https://user-images.githubusercontent.com/26118556/119977064-94c7e680-bfd5-11eb-8398-e8af07db280c.png)

**Comment by pranittare** [comment]
@arik555 Thats works perfectly!

**Comment by dovranJorayev** [comment]
> @jpuri @Sanjay007 Hello. We can do it using CSS.
> 
> **The code is given below.**
> 
> .rdw-image-alignment { display: inline !important; }
> 
> .rdw-image-center { display: flex !important; justify-content: center; }
> 
> .rdw-image-right { float: right !important; }
> 
> .rdw-image-left { float: left !important; }
> 
> ![image](https://user-images.githubusercontent.com/26118556/119977263-d5276480-bfd5-11eb-8116-32e6733ebed2.png)

How it will affect to html string  saved by editor ?
